### ISSUE TITLE
OS crash when running create_install_iso.sh under QEMU

### ISSUE THREAD

**Issue by CyberShadow** [kholia/OSX-KVM#66]
My real Apple laptop's HDD died, but I got an older version of OS X (10.6) running in qemu.

However, when running `create_install_iso.sh`, the system crashes:

![](https://dump.thecybershadow.net/4a74d1ab922310c0cce21e43af10dcaf/Screenshot_macos-10.6_2017-03-30_15%3A16%3A29.png)

Any ideas?

**Comment by CyberShadow** [comment]
Using method 3 (non-root) results in an error:

![](https://dump.thecybershadow.net/45101431d4ecd56cc2bff98f0bcfe383/Screenshot_macos-10.6_2017-03-30_15%3A25%3A48.png)

**Comment by kholia** [comment]
Try using a prebuilt image from https://bit.do/bootable.

**Comment by CyberShadow** [comment]
> Try using a prebuilt image from https://bit.do/bootable.

Thanks, but even after creating an account, I can't complete any of the downloads, due to "Your download cannot proceed, because it would take you over the current free transfer allowance for your IP address.".

**Comment by kholia** [comment]
Tough situation. 

1. Ask a friend with a real macOS system to create an ISO image for you.

2. Request other users to create a torrent for the ISO image(s). I don't have the bandwidth to do so myself. However, don't open a new issue for this.

3. Buy a real macOS system yourself.

**Comment by CyberShadow** [comment]
Thanks. I left it overnight and the download eventually completed.

However, since (as I understood) these images are not bootable, I'd still need a working OS X instance to run the conversion script, which put me back in square one.

In the end I asked a friend to lend me their work MacBook, which finally worked.

**Comment by kholia** [comment]
Great! 👍 

1. Images available at https://bit.do/bootable are bootable, and they work great with Enoch bootloader.

2. Enoch is open-source and the source code can be found [here](http://forge.voodooprojects.org/p/chameleon/source/changes/HEAD/).

**Comment by CyberShadow** [comment]
> Images available at https://bit.do/bootable are bootable, and they work great with Enoch bootloader.

Aah. Yes, I've checked and that's true, my bad. I had only tried it with Gabriel's VFIO fork; additionally, I was misled by the README file in the same directory stating that the images are "untouched" (I understand that the InstallESD.dmg images as obtained from the App Store are not bootable, and must be prepared to make them bootable).

> Enoch is open-source and the source code can be found here.

I stand corrected, thanks! I made a mistake as I did not see anything mentioning the source code on its [forum thread](http://www.insanelymac.com/forum/files/file/71-enoch/).
